### ISSUE TITLE
Too small menu on screen with big resolution

### ISSUE THREAD

**Issue by StankaCeki** [PoloShock/psdiagram#3]
When I start the application PS diagram on my notebook, it is too small to read (Picture). 
![screen_1](https://user-images.githubusercontent.com/73297499/96884862-47edc180-1482-11eb-92e5-27016ac42a41.png)
My resolution is 3840 x 2160. 

When I change it to 1920 x 1080, the application menu changed and its readable. (Picture) 
![screen_2](https://user-images.githubusercontent.com/73297499/96885423-dc582400-1482-11eb-8aef-d5e0e959b2f3.png)

Can this be fixed? Is there any workaround ? 
Thank you.

**Comment by PoloShock** [comment]
Hi, 
thank you for opening the issue. I never tested it on 4K resolution so I did not know there was an issue. I just tested it at work where I have access to 4K display and I must confirm it looks the same as in picture you posted.

From reading about the issue [here](https://stackoverflow.com/questions/15659044/how-to-set-the-dpi-of-java-swing-apps-on-windows-linux) and [here](https://superuser.com/questions/988379/how-do-i-run-java-apps-upscaled-on-a-high-dpi-display) it should be possible to deal with higher resolutions in Java Swing GUIs from Java 9. PS Diagram currently runs on Java 8.

> Can this be fixed? Is there any workaround ?

I believe it can be fixed once I upgrade the embedded Java to version >= 9. I don't think there is any workaround, unfortunately (except lowering the resolution as you did).

I will keep this issue open and once I find some time, I will look into it with priority.
